doc/mgr/orchestrator.rst

```diff
@@ -144,6 +144,11 @@ Example command::
 
      ceph orch device zap my_hostname /dev/sdx
 
+.. note::
+    If you use the ``all-available-devices`` option when creating OSD's, when you ``zap`` a device, cephadm orchestrator will automatically create a new OSD in the device .
```

Comment:
This behavior is not bound to the `all-available-devices` option. It's rather bound to the absence of a `unmanaged` flag. Id rephrase it like so:

`cephadm` will automatically deploy drives that match the `DriveGroup` in your `OSDSpec` if the `unmamanged` flag is unset. 
.. And then a sentence about how a `zap` would leave the disk `available` and would be picked up by cephadm again.